muted-tests.yml

```diff
@@ -581,27 +581,6 @@ tests:
 - class: org.elasticsearch.multiproject.test.CoreWithMultipleProjectsClientYamlTestSuiteIT
   method: test {yaml=indices.resolve_index/10_basic_resolve_index/Resolve index with hidden and closed indices}
   issue: https://github.com/elastic/elasticsearch/issues/130568
```

Comment:
Let's also remove the mutes for:

- class: org.elasticsearch.common.ssl.DefaultJdkTrustConfigTests
  method: testGetSystemTrustStoreWithNoSystemProperties
  issue: https://github.com/elastic/elasticsearch/issues/130517
- class: org.elasticsearch.common.ssl.DefaultJdkTrustConfigTests
  method: testGetNonPKCS11TrustStoreWithPasswordSet
  issue: https://github.com/elastic/elasticsearch/issues/130519

And add the issues to the closed ones -- I just tested and this fixes those failures, too.